features/upgrade/storage/upgrade.feature

```diff
@@ -191,10 +191,18 @@ Feature: Storage upgrade tests
     When I run the :apply client command with:
       | f | csi-rbac.yaml | 
     Then the step should succeed
-    Given SCC "privileged" is added to the "csi-provisioner" service account
-    Given SCC "privileged" is added to the "csi-attacher" service account
-    Given SCC "privileged" is added to the "csi-snapshotter" service account
-    Given SCC "privileged" is added to the "csi-plugin" service account
+    When I run the :oadm_policy_add_scc_to_user admin command with:
+      | scc           | privileged      |
+      |serviceaccount | csi-provisioner | 
```

Comment:
a space between `|` and `serviceaccount` is needed... pls apply to the rest of the scenario.